installer/functions

```diff
@@ -239,3 +239,60 @@ websocketcommand() {
         echo "EError timeout"
     fi
 }
+
+# Check space requirements on CHROOTS partition
+# $1: the partition's minimum size threshold
+# $2: the partition's minimum available space threshold
+check_space() {
+    local part=${CHROOTS:-/usr/local}
+    df -m $part | awk 'FNR == 2 {print $2,$4,$6}' |\
+    while read -r size avail mp; do
+        if [ "$size" -lt "$1" ]; then
+            local size_gb="$((${size}/1000))GB"
+            echo "WARNING: Your CHROOTS partition is rather small (${size_gb})."
+            echo "WARNING: Usually $((${1}/1000))GB or more is needed. "
+        #   error 1 "ERROR: Choose another partition then try again..."
+            return 1
+        elif [ "$avail" -lt "$2" ]; then
+            local avail_gb="$((${avail}/1000))GB"
+            echo "WARNING: Your available space for CHROOTS is rather small (${avail_gb})."
+            echo "WARNING: Usually $((${2}/1000))GB or more is needed. "
+        #   error 1 "ERROR: Free up more space then try again..."
+            return 2
+        else
+        #   echo "Your space requirements for CHROOTS have been met."
+            return 0
+        fi
+    done
+}
+
+# Check if partition 7 / ROOT-C (ChrUbuntu) has been resized
```

Comment:
Seeing how ChrUbuntu is mostly being superseded by other projects, it's probably worth changing references to just "dual boot"